### PR TITLE
Grayscale Integration logos when ignored

### LINKED ISSUE #5667: Use grayscale filter on logos for ignored integrations
## The request

This is how an ignored and a discovered integration look currently side by side. Although the ignored one has gray coloring at the top to differentiate it, I think your initial focal point will still be on the brand logos.

image

So I had an idea to make the brand logo for **ignored integrations** display in [grayscale](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/grayscale). In this below mockup you should be more easily able to differentiate these at a quick glance.

image

I realize that most people (including me) will probably just use the "Hide ignored integrations" option and rarely see these side by side. But when you do have that "Show" option on, this adds just a tiny bit more polish imo.

### PR DESCRIPTION
**Pull request by matstrange**
## Proposed change

This change is to render logos in grayscale if the _Discovered_ integration has been _Ignored_ - see Feature Request #5667 . This is to detract attention from the logo while the integration is in this state.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #5667 
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by matstrange** [comment]
I only have zeroconf devices on my network so I have only partially tested this since you cannot ignore those - I applied the styling to non ignore-able entity but then moved the code across.

Any suggestion on how to simulate this for discoverables?

File: src/panels/config/integrations/ha-config-integrations.ts [hunk 1]
```diff
@@ -619,6 +620,9 @@ class HaConfigIntegrations extends SubscribeMixin(LitElement) {
         .ignored {
           border: 1px solid var(--light-theme-disabled-color);
         }
+        .ignored-image {
```
Comment (bramkragten) [inline src/panels/config/integrations/ha-config-integrations.ts:623]:
```suggestion
        .ignored img {
```

File: src/panels/config/integrations/ha-config-integrations.ts [hunk 2]
```diff
@@ -327,6 +327,7 @@ class HaConfigIntegrations extends SubscribeMixin(LitElement) {
                         <img
                           src="https://brands.home-assistant.io/${item.domain}/logo.png"
                           referrerpolicy="no-referrer"
+                          class="ignored-image"
```
Comment (bramkragten) [inline src/panels/config/integrations/ha-config-integrations.ts:330]:
```suggestion
```

**Comment by bramkragten** [comment]
> I only have zeroconf devices on my network so I have only partially tested this since you cannot ignore those - I applied the styling to non ignore-able entity but then moved the code across.
> 
> Any suggestion on how to simulate this for discoverables?

You could add the styling to `.discovered img`

**Comment by matstrange** [comment]
Tested as suggested on `.discovered img` and it worked as intended. 

Thanks for the suggestions, still a newbie at git/HA contributing

**Review by bramkragten — APPROVED** [review]

**Pull request merged**